From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 5 Mar 2024 16:55:08 -0400
Subject: Disable sphinxcontrib-towncrier

The towncrier config is not shipped in the pip source releases, so
there's no point trying to use it. Not needed, anyway, as we are only
shipping final releases, where the changelog is already serialized out
of towncrier.

Forwarded: not-needed
---
 docs/html/conf.py  | 2 +-
 docs/html/news.rst | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/html/conf.py b/docs/html/conf.py
index 0a5cdbc..4154d38 100644
--- a/docs/html/conf.py
+++ b/docs/html/conf.py
@@ -32,7 +32,7 @@ if "man" not in tags:  # type: ignore[name-defined] # noqa: F821
             "myst_parser",
             "sphinx_copybutton",
             "sphinx_inline_tabs",
-            "sphinxcontrib.towncrier",
+            #"sphinxcontrib.towncrier",
             "sphinx_issues",
         ),
     )
diff --git a/docs/html/news.rst b/docs/html/news.rst
index af1d104..5769224 100644
--- a/docs/html/news.rst
+++ b/docs/html/news.rst
@@ -7,6 +7,6 @@ Changelog
     Major and minor releases of pip also include changes listed within
     prior beta releases.
 
-.. towncrier-draft-entries:: Not yet released
+.. .. towncrier-draft-entries:: Not yet released
 
 .. pip-news-include:: ../../NEWS.rst
